tools/spelling/spelling_dictionary.txt

```diff
@@ -176,6 +176,7 @@ ITOA
 Injectable
 Isode
 Iters
+JA
```

Comment:
consider using backticks around this in the comment, instead of adding to the dictionary